internal/policy/policy.go

```diff
@@ -15,29 +12,10 @@ const (
 
 // InitializeNamespace creates a git ref for the policy. Initially, the entry
 // has a zero hash.
-// Note: policy.InitializeNamespace assumes the gittuf namespace has been
-// created already.
-func InitializeNamespace() error {
-	repoRootDir, err := common.GetRepositoryRootDirectory()
-	if err != nil {
+func InitializeNamespace(repo *git.Repository) error {
+	if err := repo.Storer.SetReference(plumbing.NewHashReference(plumbing.ReferenceName(PolicyRef), plumbing.ZeroHash)); err != nil {
```

Comment:
Should we add a check here to ensure we don't re-init to zero?

Reply:
Or does it belong in `Repository.InitializeNamespaces`?

Reply:
Sounds like a good idea to me! I'd lean towards checking here, since this is exported callers could call this directly, and checking here would also cover InitializeNamespaces.

Reply:
Done! Added a test to RSL and policy that checks for this.